packages/react-cosmos-router-proxy/src/LocationInterceptor.js

```diff
@@ -0,0 +1,22 @@
+import { Component } from 'react';
+import { object } from 'prop-types';
+
+// Heads up: This is tied to React Router internals and is subject to change
+const getPathFromContext = context => context.router.route.location.pathname;
```

Comment:
Not enjoying this, being an internal API it could change whenever the maintainers feel like it 😢